quantumdb-postgresql/src/test/java/io/quantumdb/core/backends/integration/multistate/MultiStateTest.java

```diff
@@ -50,29 +50,30 @@ public void before() throws SQLException, ClassNotFoundException, MigrationExcep
 		step0 = changelog.getRoot();
 
 		step1 = changelog.addChangeSet("step1", "Michael de Jong", "Create test table.",
-				createTable("test").with("id", bigint(), PRIMARY_KEY, AUTO_INCREMENT))
+						createTable("test").with("id", bigint(), PRIMARY_KEY, AUTO_INCREMENT))
```

Comment:
4 tabs seems a little excessive.

Reply:
IntelliJ automatically does this, I have manually deleted those tabs